deployment/cloud-config.yml

```diff
@@ -0,0 +1,27 @@
+#cloud-config
+mounts:
+- [ /dev/nvme0n1p1, /mnt/data/ ]
```

Comment:
Just to make sure I'm following correctly, `/dev/nvme0n1p1` is the device name given to the snapshot you're wiring up via the block device mappings?

Reply:
The device name given to the snapshot in the block device mappings is `/dev/sdf`. However, T3 instances are built on the Nitro System, so EBS volumes are exposed as NVMe block devices.

So, yeah, I determined `/dev/nvme0n1p1` corresponds to the EBS volume.

If we were running Amazon Linux, it would create a symbolic link from the device name in the block device mapping (for example, `/dev/sdf`), to the NVMe device name.